src/views/AnnualReport.vue

```diff
@@ -174,7 +174,6 @@
                 <ar-date />
                 <br>
                 <summary-office-addresses
-                  v-if="originalAddresses"
```

Comment:
No longer needed as "originalAddress" is now initialized in a way that the next 2 lines can de-reference it (which is what failed previously).